### ISSUE TITLE
Software Updates + Backups Calendar

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop#4510]
## Description
Newsroom admins too rarely conduct update and backup maintenance on the recommended schedule. To reduce the burden of maintaining instances, FPF should create an Instance Maintenance Calendar admins can subscribe to via their calendaring software, with all updates handily keyed-in to alert admins when the time(s) for them to act are.

Once created, this calendar would also need to be maintained—so its lifecycle governance would also be a required acceptance criteria, beyond whatever touchpoint(s) may be created for newsrooms.

## User Research Evidence
From David's UX Meeting presentation: https://github.com/freedomofpress/securedrop-ux/wiki/UX-meeting-20190523
- Updates happen too infrequently
    - "many cycles lost in training, to just updating hardware"
    - "Could we surface an update calendar or the need to update more often, in docs?"
    - Could we create a maintenence calendar in GCal for Admins to follow?
    - Could we create something for Firefox to help with this?
    - ^ If calendar created, including backup reminders wd be good, too!

## Source Materials
FPF team members should be engaged to meet with any volunteers interested in tackling this Issue. The materials exist to guide the Issue; they're just in many places, and a 30min phone meeting to discuss the needs around this would likely be more helpful. 

Alternately, if @eloquence or @huertanix or @zenmonkeykstop  or others would like to collaborate on assembling one, clean comment to this Issue that spells-out each backup/update need to include on that calendar with links-out to external relevant schedules (such as the TAILS release cycle), that could probably work, too.

**Comment by eloquence** [comment]
I agree this could be very helpful, and we could promote it in our release announcements, social media, and on-boarding documentation for administrators. I think this could be a good task for the Newsroom Services Coordinator at FPF to take on in the near future, once that position is filled. We could maintain it for a year and include an "Are you using this / is it helpful" question in the next admin survey.
